src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -341,4 +346,3 @@
        :width    "100%"
        :height   "110%"}}
```

Comment:
I think these values are old, so probably you need to rebase this branch :)

Reply:
yeah